include/eve/module/core/regular/is_equal.hpp

```diff
@@ -57,53 +57,47 @@ namespace eve
 //!   @code
 //!   namespace eve
 //!   {
-//!      template< eve::value T, eve::value U >
-//!      auto is_equal(T x, U y) noexcept;
+//!      // Regular overload
+//!      constexpr auto is_equal(value auto x, value auto y) noexcept;                          // 1
+//!
+//!      // Lanes masking
+//!      constexpr auto is_equal[conditional_expr auto c](value auto x, value auto y) noexcept; // 2
+//!      constexpr auto is_equal[logical_value auto m](value auto x, value auto y) noexcept;    // 2
+//!
+//!      // Semantic option
+//!      constexpr auto is_equal[numeric](/*any of the above overloads*/)             noexcept; // 3
+//!      constexpr auto is_equal[almost](/*any of the above overloads*/)              noexcept; // 4
+//!      constexpr auto is_equal[almost = tol](/*any of the above overloads*/)        noexcept; // 4
 //!   }
 //!   @endcode
 //!
 //!   **Parameters**
 //!
-//!     * `x`, `y` :  arguments
+//!     * `x`, `y`:  [arguments](@ref eve::value).
+//!     * `c`: [Conditional expression](@ref conditional_expr) masking the operation.
+//!     * `m`: [Logical value](@ref logical) masking the operation.
+//!     * `tol`: [scalar value](@ref value) tolerance.
 //!
 //!   **Return value**
 //!
-//!     Returns the logical value containing the [elementwise](@ref glossary_elementwise) equality
-//!     test result between `x` and `y`. The infix notation `x == y` can also be used.
-//!
-//!   @note
-//!      Although the infix notation with `==` is supported, the `==` operator on
+//!      1. Returns the logical value containing the [elementwise](@ref glossary_elementwise) equality
+//!        test result between `x` and `y`. The infix notation `x == y` can also be used.
+//!      2. [The operation is performed conditionnaly](@ref conditional).
+//!      3. The expression `is_equal[numeric](x,y)` considers that `Nan` values are equal.
+//!      4. The expression `is_equal[almost = tol](x, y)` where `x` and `y` must be
+//!         floating point values, evals to true if and only if `x` is almost equal to `y`.
```

Comment:
evas is nto a verb. Evaluates is. Please change everywhere

Reply:
done

Reply:
ok